examples/cordova-sample/MyApp/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "cordova": "^10.0.0",
     "cordova-annotated-plugin-android": "^1.0.4",
-    "cordova-plugin-purchases": "file:../../..",
+    "cordova-plugin-purchases": "file:../../../../cordova-plugin-purchases",
```

Comment:
Do we want to commit this? This looks like it assuming the name of our root directory for this git repo which... is probably fine like 99% of the time but 🤷‍♂️

Reply:
I think we do, this gets edited when running `bin/setup.sh` and since it's the default I think we should go with it. Can you double check? I just reverted the file and after running setup.sh, it came back 🤷‍♂️

Reply:
Gonna merge, but we can revisit if needed.